livekit-agents/livekit/agents/voice/agent_activity.py

```diff
@@ -1080,35 +1080,33 @@ def _on_first_frame(_: asyncio.Future) -> None:
         if speech_handle.interrupted:
             await utils.aio.cancel_and_wait(*tasks)
 
+            forwarded_text = text_out.text
```

Comment:
should we simplify by starting the state to `""`

This would simplify the branch below, (just moving the else branch to ` if playback_ev.synchronized_transcript is not None:`

nit tho